code/modules/library/skill_books.dm

```diff
@@ -0,0 +1,617 @@
+
+//piloting
+/obj/item/book/skill/general/pilot
+	skill = SKILL_PILOT
+	author = "Sumi Shimamoto"
+
+/obj/item/book/skill/general/pilot/basic
+	skill_req = SKILL_NONE
+	name = "beginner piloting textbook"
+
+/obj/item/book/skill/general/pilot/adept
+	skill_req = SKILL_BASIC
+	name = "intermediate piloting textbook"
+
+/obj/item/book/skill/general/pilot/expert
+	skill_req = SKILL_ADEPT
+	name = "advanced piloting textbook"
+
+/obj/item/book/skill/general/pilot/prof
+	skill_req = SKILL_EXPERT
+	name = "theoretical piloting textbook"
+
+//hauling
+/obj/item/book/skill/general/hauling
+	skill = SKILL_HAULING
+	author = "Chiel Brunt"
+
+/obj/item/book/skill/general/hauling/basic
+	skill_req = SKILL_NONE
+	name = "beginner athletics textbook"
+
+/obj/item/book/skill/general/hauling/adept
+	skill_req = SKILL_BASIC
+	name = "intermediate athletics textbook"
+
+/obj/item/book/skill/general/hauling/expert
+	skill_req = SKILL_ADEPT
+	name = "advanced athletics textbook"
+
+/obj/item/book/skill/general/hauling/prof
+	skill_req = SKILL_EXPERT
+	name = "theoretical athletics textbook"
+
+//computer
+/obj/item/book/skill/general/computer
+	skill = SKILL_COMPUTER
+	author = "Simona Castiglione"
+
+/obj/item/book/skill/general/computer/basic
+	skill_req = SKILL_NONE
+	name = "beginner information technology textbook"
+
+/obj/item/book/skill/general/computer/adept
+	skill_req = SKILL_BASIC
+	name = "intermediate information technology textbook"
+
+/obj/item/book/skill/general/computer/expert
+	skill_req = SKILL_ADEPT
+	name = "advanced information technology textbook"
+
+/obj/item/book/skill/general/computer/prof
+	skill_req = SKILL_EXPERT
+	name = "theoretical information technology textbook"
+
+/* 
+SERVICE
+*/
+/obj/item/book/skill/service
+
+//botany
+/obj/item/book/skill/service/botany
+	icon_state = "bookHydroponicsPodPeople"
+	skill = SKILL_BOTANY
+	author = "Mai Dong Chat"
+
+/obj/item/book/skill/service/botany/basic
+	skill_req = SKILL_NONE
+	name = "beginner botany textbook"
+
+/obj/item/book/skill/service/botany/adept
+	skill_req = SKILL_BASIC
+	name = "intermediate botany textbook"
+
+/obj/item/book/skill/service/botany/expert
+	skill_req = SKILL_ADEPT
+	name = "advanced botany textbook"
+
+/obj/item/book/skill/service/botany/prof
+	skill_req = SKILL_EXPERT
+	name = "theoretical botany textbook"
+
+//cooking
+/obj/item/book/skill/service/cooking
+	icon_state = "barbook"
+	skill = SKILL_COOKING
+	author = "Lavinia Burrows"
+
+/obj/item/book/skill/service/cooking/basic
+	skill_req = SKILL_NONE
+	name = "beginner cooking textbook"
+
+/obj/item/book/skill/service/cooking/adept
+	skill_req = SKILL_BASIC
+	name = "intermediate cooking textbook"
+
+/obj/item/book/skill/service/cooking/expert
+	skill_req = SKILL_ADEPT
+	name = "advanced cooking textbook"
+
+/obj/item/book/skill/service/cooking/prof
+	skill_req = SKILL_EXPERT
+	name = "theoretical cooking textbook"
+
+/* 
+SECURITY
+*/
+/obj/item/book/skill/security
```

Comment:
Missing an icon set like the others have?

Reply:
I need new icons for all of them tbh. But I hate making icons.

Reply:
Made a few new icons for some of them, good enough for government work.